<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <item>
     <widget class="QPlainTextEdit" name="TextEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_E">
    <property name="title">
     <string>编辑(&amp;E)</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionFind"/>
    <addaction name="actionReplace"/>
    <addaction name="actionSelectAll"/>
   </widget>
   <widget class="QMenu" name="menu_O">
    <property name="title">
     <string>格式(&amp;O)</string>
    </property>
    <addaction name="actionFont"/>
    <addaction name="actionFontColor"/>
    <addaction name="actionEditorColor"/>
    <addaction name="actionBackgroundColor"/>
    <addaction name="actionLineNumber"/>
    <addaction name="actionLineWrap"/>
   </widget>
   <widget class="QMenu" name="menu_V">
    <property name="title">
     <string>查看(&amp;V)</string>
    </property>
    <addaction name="actionToolBar"/>
    <addaction name="actionStatusBar"/>
   </widget>
   <widget class="QMenu" name="menu_H">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_E"/>
   <addaction name="menu_O"/>
   <addaction name="menu_V"/>
   <addaction name="menu_H"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionSaveAs"/>
   <addaction name="separator"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="actionCut"/>
   <addaction name="actionCopy"/>
   <addaction name="actionPaste"/>
   <addaction name="actionFind"/>
   <addaction name="actionReplace"/>
   <addaction name="separator"/>
   <addaction name="actionFont"/>
   <addaction name="actionLineWrap"/>
   <addaction name="actionLineNumber"/>
   <addaction name="actionFontColor"/>
   <addaction name="actionEditorColor"/>
   <addaction name="actionBackgroundColor"/>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/create_new_folder_24dp_1F1F1F_FILL0_wght400_GRAD0_opsz24.png</normaloff>:/img/create_new_folder_24dp_1F1F1F_FILL0_wght400_GRAD0_opsz24.png</iconset>
   </property>
   <property name="text">
    <string>新建(&amp;N)</string>
   </property>
   <property name="toolTip">
    <string>新建</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/file_open_24dp_1F1F1F_FILL0_wght400_GRAD0_opsz24.png</normaloff>:/img/file_open_24dp_1F1F1F_FILL0_wght400_GRAD0_opsz24.png</iconset>
   </property>
   <property name="text">
    <string>打开(&amp;O)</string>
   </property>
   <property name="toolTip">
    <string>打开</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/save_24dp_1F1F1F_FILL0_wght400_GRAD0_opsz24.png</normaloff>:/img/save_24dp_1F1F1F_FILL0_wght400_GRAD0_opsz24.png</iconset>
   </property>
   <property name="text">
    <string>保存(&amp;S)</string>
   </property>
   <property name="toolTip">
    <string>保存</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/save_as_24dp_1F1F1F_FILL0_wght400_GRAD0_opsz24.png</normaloff>:/img/save_as_24dp_1F1F1F_FILL0_wght400_GRAD0_opsz24.png</iconset>
   </property>
   <property name="text">
    <string>另存为(&amp;A)</string>
   </property>
   <property name="toolTip">
    <string>另存为</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/exit_to_app_24dp_1F1F1F_FILL0_wght400_GRAD0_opsz24.png</normaloff>:/img/exit_to_app_24dp_1F1F1F_FILL0_wght400_GRAD0_opsz24.png</iconset>
   </property>
   <property name="text">
    <string>退出(&amp;X)</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/undo_24dp_1F1F1F_FILL0_wght400_GRAD0_opsz24.png</normaloff>:/img/undo_24dp_1F1F1F_FILL0_wght400_GRAD0_opsz24.png</iconset>
   </property>
   <property name="text">
    <string>撤销(&amp;U)</string>
   </property>
   <property name="toolTip">
    <string>撤销</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/content_cut_24dp_1F1F1F_FILL0_wght400_GRAD0_opsz24.png</normaloff>:/img/content_cut_24dp_1F1F1F_FILL0_wght400_GRAD0_opsz24.png</iconset>
   </property>
   <property name="text">
    <string>剪切(&amp;T)</string>
   </property>
   <property name="toolTip">
    <string>剪切</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/content_copy_24dp_1F1F1F_FILL0_wght400_GRAD0_opsz24.png</normaloff>:/img/content_copy_24dp_1F1F1F_FILL0_wght400_GRAD0_opsz24.png</iconset>
   </property>
   <property name="text">
    <string>复制(&amp;C)</string>
   </property>
   <property name="toolTip">
    <string>复制</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/content_paste_24dp_1F1F1F_FILL0_wght400_GRAD0_opsz24.png</normaloff>:/img/content_paste_24dp_1F1F1F_FILL0_wght400_GRAD0_opsz24.png</iconset>
   </property>
   <property name="text">
    <string>粘贴(&amp;P)</string>
   </property>
   <property name="toolTip">
    <string>粘贴</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionFind">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/find_in_page_24dp_1F1F1F_FILL0_wght400_GRAD0_opsz24.png</normaloff>:/img/find_in_page_24dp_1F1F1F_FILL0_wght400_GRAD0_opsz24.png</iconset>
   </property>
   <property name="text">
    <string>查找(&amp;F)</string>
   </property>
   <property name="toolTip">
    <string>查找</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionReplace">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/find_replace_24dp_1F1F1F_FILL0_wght400_GRAD0_opsz24.png</normaloff>:/img/find_replace_24dp_1F1F1F_FILL0_wght400_GRAD0_opsz24.png</iconset>
   </property>
   <property name="text">
    <string>替换(&amp;R)</string>
   </property>
   <property name="toolTip">
    <string>替换</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionFont">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/font_download_24dp_1F1F1F_FILL0_wght400_GRAD0_opsz24.png</normaloff>:/img/font_download_24dp_1F1F1F_FILL0_wght400_GRAD0_opsz24.png</iconset>
   </property>
   <property name="text">
    <string>字体(&amp;F)</string>
   </property>
   <property name="toolTip">
    <string>字体</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionLineWrap">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/wrap_text_24dp_1F1F1F_FILL0_wght400_GRAD0_opsz24.png</normaloff>:/img/wrap_text_24dp_1F1F1F_FILL0_wght400_GRAD0_opsz24.png</iconset>
   </property>
   <property name="text">
    <string>自动换行(&amp;W)</string>
   </property>
   <property name="toolTip">
    <string>自动换行</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionToolBar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/toolbar_24dp_1F1F1F_FILL0_wght400_GRAD0_opsz24.png</normaloff>:/img/toolbar_24dp_1F1F1F_FILL0_wght400_GRAD0_opsz24.png</iconset>
   </property>
   <property name="text">
    <string>工具栏(&amp;T)</string>
   </property>
   <property name="toolTip">
    <string>工具栏</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionStatusBar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/analytics_24dp_1F1F1F_FILL0_wght400_GRAD0_opsz24.png</normaloff>:/img/analytics_24dp_1F1F1F_FILL0_wght400_GRAD0_opsz24.png</iconset>
   </property>
   <property name="text">
    <string>状态栏(&amp;S)</string>
   </property>
   <property name="toolTip">
    <string>状态栏</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/redo_24dp_1F1F1F_FILL0_wght400_GRAD0_opsz24.png</normaloff>:/img/redo_24dp_1F1F1F_FILL0_wght400_GRAD0_opsz24.png</iconset>
   </property>
   <property name="text">
    <string>恢复(&amp;)</string>
   </property>
   <property name="toolTip">
    <string>恢复</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionSelectAll">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/select_all_24dp_1F1F1F_FILL0_wght400_GRAD0_opsz24.png</normaloff>:/img/select_all_24dp_1F1F1F_FILL0_wght400_GRAD0_opsz24.png</iconset>
   </property>
   <property name="text">
    <string>全选(&amp;A)</string>
   </property>
   <property name="toolTip">
    <string>全选</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/info_24dp_1F1F1F_FILL0_wght400_GRAD0_opsz24.png</normaloff>:/img/info_24dp_1F1F1F_FILL0_wght400_GRAD0_opsz24.png</iconset>
   </property>
   <property name="text">
    <string>关于(&amp;A)</string>
   </property>
   <property name="toolTip">
    <string>关于</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionLineNumber">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/numbers_24dp_1F1F1F_FILL0_wght400_GRAD0_opsz24.png</normaloff>:/img/numbers_24dp_1F1F1F_FILL0_wght400_GRAD0_opsz24.png</iconset>
   </property>
   <property name="text">
    <string>显示行号(&amp;L)</string>
   </property>
   <property name="toolTip">
    <string>显示行号</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionFontColor">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/format_color_text_24dp_1F1F1F_FILL0_wght400_GRAD0_opsz24.png</normaloff>:/img/format_color_text_24dp_1F1F1F_FILL0_wght400_GRAD0_opsz24.png</iconset>
   </property>
   <property name="text">
    <string>字体颜色</string>
   </property>
   <property name="toolTip">
    <string>字体颜色</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionEditorColor">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/palette_24dp_1F1F1F_FILL0_wght400_GRAD0_opsz24.png</normaloff>:/img/palette_24dp_1F1F1F_FILL0_wght400_GRAD0_opsz24.png</iconset>
   </property>
   <property name="text">
    <string>编辑器背景色</string>
   </property>
   <property name="toolTip">
    <string>编辑器背景色</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionBackgroundColor">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/format_color_fill_24dp_1F1F1F_FILL0_wght400_GRAD0_opsz24.png</normaloff>:/img/format_color_fill_24dp_1F1F1F_FILL0_wght400_GRAD0_opsz24.png</iconset>
   </property>
   <property name="text">
    <string>字体背景色</string>
   </property>
   <property name="toolTip">
    <string>字体背景色</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
